### DISCUSSION TITLE
Welcome to efficientdet-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by rwightman** [rwightman/efficientdet-pytorch#134] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about, especially those not suited for Issues. Issues will be exclusively focused on verifiable bugs and feature requests that benefit all users of the library.
  * Share ideas.
  * Engage with other community members.
  * Respect others. This is a community.

**Comment by Ekta246** [comment]
I see the Lr scheduler.step_update takes the eval_metric as mAP set to default in the train.py
However, when I use the eval_metric mAP I see the validation loss diverges in a weird manner.

While, when I change the eval_metric to Loss in the LR scheduler update, I see the model has a tremendous improvement learning and the validation loss converges.

 https://github.com/rwightman/efficientdet-pytorch/blob/611532db49fdd691f48f913bc433391a12014bd8/train.py#L416

However, I believe there is a bug in my mAP evaluation maybe.

Also, I calculate mean Intersection of Union on the validation set and the test set. It is pretty good. It is around 90%
The predictions that the model generates are pretty close to the ground truth. So, it's a good news.
